obs-websocket-dotnet/OBSWebsocket.cs

```diff
@@ -320,6 +320,9 @@ public OBSWebsocket()
         /// <param name="password">Server password</param>
         public void Connect(string url, string password)
         {
+            if (url.ToLower().StartsWith("ws://") == false)
+                url = "ws://" + url;
```

Comment:
I would just generate a clear exception vs adding this to the URL.
Something like `new AuthFailureException(@"Invalid server url, must start with 'ws://'");`